cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -1103,7 +1139,44 @@ func newServer(conf rootConfig) (*Server, error) {
 }
 
 func main() {
-	if err := rootCmd().ParseAndRun(context.Background(), os.Args[1:]); err != nil {
+	cmd := rootCmd()
+	if err := cmd.Parse(os.Args[1:]); err != nil {
+		log.Fatal(err)
+	}
+
+	debugFlagOverride := false
+
+	flag := cmd.FlagSet.Lookup("debug")
+	if debugFlag, err := strconv.ParseBool(flag.Value.String()); err == nil {
+		// Debug flag overrides a non-debug logging level
+		debugFlagOverride = debugFlag && !srcLogLevelIsDebug()
```

Comment:
I reckon just override the envvar if the flag is passed in. You don't then need to later check if we overode it (should simplify invariants / conditionals.

Reply:
resolved; no longer using debug flag